torch/lib/THPP/tensors/generic/THTensor.cpp

```diff
@@ -113,7 +122,7 @@ auto THTensor<real>::resize(THLongStorage *size,
 
 template<>
 auto THTensor<real>::resizeAs(const Tensor& src) -> THTensor& {
-  THTensor_(resizeAs)(tensor, dynamic_cast<const THTensor<real>&>(src).tensor);
```

Comment:
It is possible to `dynamic_cast` from const Base class to const Derived class. Since we do not modify reference or its internal members (`.tensor` internal members can be modified though) so it is legal and possible to do without dropping `const`.